code/_onclick/hud/robot.dm

```diff
@@ -1,241 +1,241 @@
-/obj/screen/robot
```

Comment:
I have no idea why this happened. It deletes and readds everything with no differences, and I never even opened this file.